emb/network/transport/pico/serial.cc

```diff
@@ -25,21 +25,27 @@ void Serial::send(const std::span<uint8_t> &data) {
 }
 
 std::span<uint8_t> Serial::receive(std::span<uint8_t> buffer) {
-    int16_t rc = getchar_timeout_us(1000);
-    uint16_t rx_size = 0;
-    while (rc != PICO_ERROR_TIMEOUT) {
-        buffer[rx_size++] = rc;
+    int16_t rc = getchar_timeout_us(10'000);
```

Comment:
Realizing my laptop had significant delays while writing large buffers was Big Sad (TM) to debug.